zk/server_help.go

```diff
@@ -39,6 +54,10 @@ func StartTestCluster(size int, stdout, stderr io.Writer) (*TestCluster, error)
 			cluster.Stop()
 		}
 	}()
+	options := &options{retryTimes: 10}
```

Comment:
Why not use WithRetryTimes() func to init

Reply:
default value, show declared variables is ok

Reply:
Fine ^_^